### DISCUSSION TITLE
question about ESP Muse Luxe

### DISCUSSION THREAD

**Discussion by userMak** [music-assistant/support#1275] [Q&A] [answered]
Hi,
I am using the following speaker 
https://raspiaudio.com/produit/esp-muse-luxe

Although it is working fine through MA, I can not change the volume from the physical buttons of the speaker.  This is a bit problematic especially for my family (they use to lower the voice manual)

When I connect to the speakers via Bluetooth of my phone for example (and not via Music Assistant) the buttons are working fine.

Is there a way that we could fix this?

**Comment by marcelveldt** [comment, marked as answer]
Yes, this is fixed in the next release (beta33)

**Reply by userMak** [reply]
Thanks!

**Comment by sofixa** [comment]
@userMak with which firmware are you using the Muse Luxe?
